elements/pl-string-input/pl-string-input.py

```diff
@@ -52,7 +51,7 @@ def render(element_html, data):
             'remove-spaces': remove_spaces,
             'editable': editable,
             'info': info,
-            'shortinfo': shortinfo,
+            'shortinfo': placeholder,
```

Comment:
Rename the `shortinfo` variable to `placeholder` to make the code easier to follow?